src/NServiceBus.Core/UnitOfWork/TransactionScopes/TransactionScopeUnitOfWorkBehavior.cs

```diff
@@ -36,7 +36,6 @@ public class Registration : RegisterStep
                 "Makes sure that the handlers gets wrapped in a transaction scope",
                 b => new TransactionScopeUnitOfWorkBehavior(transactionOptions))
             {
-                InsertAfter("ExecuteUnitOfWork");
```

Comment:
This will mean that the scope will wrap the handlers like advertised but I wonder if we still have to place it in the same place in the incoming physical pipeline?

Reply:
As discussed, seems like it should be fine anywhere in the physical stage of the pipeline.